src/binder/binder_context.cpp

```diff
@@ -50,7 +51,7 @@ bool BinderContext::GetColumnPosTuple(
     type::TypeId& value_type) {
   auto db_id = std::get<0>(table_id_tuple);
   auto table_id = std::get<1>(table_id_tuple);
-  auto schema = catalog::Catalog::GetInstance()
+  auto schema = catalog::CatalogStorageManager::GetInstance()
```

Comment:
Yes that's correct. 

I don't think that we should call this the "CatalogStorageManager".  It should just be the "StorageManager"